onebeyond/cuckoojs
# Feat/template lambda quickstart

## What’s the focus of this PR
This schematics generates a lambda project using serverless + dependency injection using awilix (this must be defined with architecture team during the next weeks, but i think it's important to start releasing things to keep pushing it and iterate)

Also, it implements changes on cli to launch the lambda quickstart among other implemented schematics. 

Note: There are some issues with the cli commands that will involve some refactors later (like commands output or commands overlapping), but in order to not being blocked and deliver value, we've created another command to launch this feature (new-lambda) in separate files.

## How to review this PR
Create your own lambda project using the schematics and test it following the readme:
- Build schematics and cli
- Launch cli command on a separate folder with the possible options (--git-provider and --skip-git-init)
- On the project created, try to run lambda tests and lambda local invoke to verify the project created is stable and works (just follow readme)

Note: If you select --skip-git-init and you're creating the project in a non-git folder it will fail when trying to install husky hooks.

Before submitting this PR, I made sure:
- [x] I'm using our guidelines
- [x] I've added unit tests if feasible
